packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -22,7 +22,7 @@
     "deliveredAriaLabel": "Message sent",
     "deliveredTooltipText": "Sent",
     "seenAriaLabel": "Message seen by others",
-    "seenTooltipText": "Seen",
+    "seenTooltipText": "Read by {messageThreadReadCount} of {messageThreadParticipantCount}",
```

Comment:
Changing the string in a big way like this might be considered behaviorally breaking -- if Contoso had their own translations (which they could), those will break (the app will throw an error with trying to do string interpolation).

We should consider introducing a new string.

Reply:
Yes, we might need a new string here - this seenTooltipText would be a fallback for the readReceipt indicator under these scenarioes:
1. when Readby 0 of 0 (sometimes we failed to get the participantNum because of permission issues, use this 'seen' string to avoid showing some weird result)
2. People might not want readReceipt details all the time, so we might want to open some flags in future PR to disable this details feature - so keep this 'seen' string makes sense

Reply:
Updated PR